src/View/UIView.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace BootstrapUI\View;
+
+use Cake\View\View;
+
+/**
+ * BootstrapUI’s  view class
+ */
+class UIView extends View
+{
+
+    public $layout = 'BootstrapUI.default';
```

Comment:
Some thing like this works?
````php
/**
 * UIView, the extended class of View one. UIView customise the $layout to the BootstrapUI's layout
 * and initialize the View with the BootstrapUI's helpers.
 *
 * To use UIView class just extend it and make sure to render parent::initialize();
 * The full example here:
 *
 * ```
 * namespace App\View;
 * 
 * use BootstrapUI\View\UIView as View;
 * 
 * class AppView extends View
 * {
 *      public function initialize()
 *      {
 *          parent::initialize();
 *      }
 * }
 * ```
 */
````